ytypes/util_test.go

```diff
@@ -242,21 +242,24 @@ func TestStringToType(t *testing.T) {
 }
 
 type allKeysListStruct struct {
-	StringKey        *string  `path:"stringKey"`
-	Int8Key          *int8    `path:"int8Key"`
-	Int16Key         *int16   `path:"int16Key"`
-	Int32Key         *int32   `path:"int32Key"`
-	Int64Key         *int64   `path:"int64Key"`
-	Uint8Key         *uint8   `path:"uint8Key"`
-	Uint16Key        *uint16  `path:"uint16Key"`
-	Uint32Key        *uint32  `path:"uint32Key"`
-	Uint64Key        *uint64  `path:"uint64Key"`
-	Decimal64Key     *float64 `path:"decimal64Key"`
-	BoolKey          *bool    `path:"boolKey"`
-	BinaryKey        Binary   `path:"binaryKey"`
-	EnumKey          EnumType `path:"key3"`
-	UnionKey         Union1   `path:"unionKey"`
-	UnionLoneTypeKey *uint32  `path:"unionLoneTypeKey"`
+	StringKey           *string  `path:"stringKey"`
+	Int8Key             *int8    `path:"int8Key"`
+	Int16Key            *int16   `path:"int16Key"`
+	Int32Key            *int32   `path:"int32Key"`
+	Int64Key            *int64   `path:"int64Key"`
+	Uint8Key            *uint8   `path:"uint8Key"`
+	Uint16Key           *uint16  `path:"uint16Key"`
+	Uint32Key           *uint32  `path:"uint32Key"`
+	Uint64Key           *uint64  `path:"uint64Key"`
+	Decimal64Key        *float64 `path:"decimal64Key"`
+	BoolKey             *bool    `path:"boolKey"`
+	BinaryKey           Binary   `path:"binaryKey"`
+	EnumKey             EnumType `path:"enumKey"`
```

Comment:
They "key3" to "enumKey" change was intentional right?

Reply:
yep, fixing a harmless typo